castervoice/apps/browser/browser.py

```diff
@@ -0,0 +1,67 @@
+#
+# __author__ = "lexxish"
+#
+from dragonfly import Choice
+
+from castervoice.apps.shared.directions import FORWARD, RIGHT, BACK, LEFT
+from castervoice.lib.dfplus.additions import IntegerRefST
+
+OPEN_NEW_WINDOW = "(new window|win new)"
+OPEN_NEW_INCOGNITO_WINDOW = "(new incognito window | incognito)"
+CLOSE_WINDOW = "win close|close all tabs"
+NEW_TAB_N_TIMES = "new tab [<n>]|tab new [<n>]"
+REOPEN_TAB_N_TIMES = "reopen tab [<n>]|tab reopen [<n>]"
+CLOSE_TAB_N_TIMES = "close tab [<n>]|tab close [<n>]"
+NEXT_TAB_N_TIMES = "%s tab [<n>]|tab %s [<n>]" % (FORWARD, RIGHT)
+PREVIOUS_TAB_N_TIMES = "%s tab [<n>]|tab %s [<n>]" % (BACK, LEFT)
+OPEN_NEW_TAB_BASED_ON_CURSOR = "new tab that"
+SWITCH_TO_TAB_N = "tab <m>|<nth> tab"
+SWITCH_TO_LAST_TAB = "last tab"
+SWITCH_TO_SECOND_TO_LAST_TAB = "second last tab"
+GO_FORWARD_N_TIMES = "go %s [<n>]" % FORWARD
+GO_BACK_N_TIMES = "go %s [<n>]" % BACK
+ZOOM_IN_N_TIMES = "zoom in [<n>]"
+ZOOM_OUT_N_TIMES = "zoom out [<n>]"
+ZOOM_RESET_DEFAULT = "zoom reset"
+FORCE_HARD_REFRESH = "(hard refresh|super refresh)"
+FIND_NEXT_MATCH = "find %s [match] [<n>]" % BACK
+FIND_PREVIOUS_MATCH = "find %s [match] [<n>]" % FORWARD
```

Comment:
back and forward should be switched here

Reply:
good catch